.gitignore

```diff
@@ -4,6 +4,7 @@
 #Compiled files.
 .sass-cache
 _site/
+_site_production/
```

Comment:
Noticed this morning that this was starting to appear as a changed file, so I added it to the gitignore. Wanted to verify this is correct.